stories/Components/Forms/Textarea/Textarea.jsx

```diff
@@ -1,16 +1,36 @@
 import React from 'react';
 
 export const Textarea = ({
-  id, type, rows, cols, placeholder, required, mode, labelText, errorText, disabled, minlength, helpText,
-}) => (
-  <>
-    <div className="input-group">
-      {labelText && <label htmlFor={[`${type}`]} className={[`input-group__label--${mode}`].join(' ')}>{ labelText }</label>}
-      <div className={[`input-group__icon ${type}-field ${mode}`].join(' ')}>
-        <textarea type={type} id={id} disabled={disabled} required={required} placeholder={placeholder} minLength={minlength} cols={cols} rows={rows} className={['input__controls', `input__controls--${type} ${mode}`].join(' ')} />
+  id, type, rows, cols, placeholder, required, labelText, errorText, minlength, helpText, State,
+}) => {
+
+  let state;
+  switch(State){
```

Comment:
@Shamimkhan321 As per @kuldev feedback we can use the following here:
```
let state;
state = State ? State.toLowerCase() : 'default';
```
or if you need to check it against certain set of states. You can also write it as:
```
let state;
const states = ["Focus", "Error", "Disabled"];
state = states.includes(State) ? State.toLowerCase() : 'default';
```